### DISCUSSION TITLE
Remove collection management from  Password Manager view?

### DISCUSSION THREAD

**Discussion by g-hodgson-tup** [dani-garcia/vaultwarden#5122] [Ideas]
Hi.  

Seeing as collection management in the Password Manager view currently has a few problems, such as:

* [not showing nested collections](https://github.com/dani-garcia/vaultwarden/discussions/4852), 
* defaulting the Organization to the first in the list, rather than the one I am currently in,

is it worth considering removing the collection management functions from the Password Manager view completely? i.e. only allowing collections to be managed in the Admin Console (where everthing appears to work fine)?

Or are you expecting the problems to be fixed in upcoming BW web app update?

Best regards,
Gary

**Comment by BlackDex** [comment]
If you use the current `testing` tagged containers you will see it works. So probably not a good idea to do that. 
Also, we do not want to change too much on the web-vault to keep as close to Bitwarden as we can

**Comment by g-hodgson-tup** [comment]
That's great to hear, and I can understand not wanting to diverge too far from the upstream web-vault!

I just tried out the testing container and can confirm the nested collections problem seems to be resolved 👍
